2.4.41/Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM centos:centos7
+
+# OpenLDAP server image for OpenShift Origin
+#
+# Volumes:
+# * /var/lib/ldap/data - Datastore for OpenLDAP
+# * /etc/openldap/     - Config directory for slapd
+# Environment:
+# * $OPENLDAP_ADMIN_PASSWORD         - OpenLDAP administrator password
+# * $OPENLDAP_DEBUG_LEVEL (Optional) - OpenLDAP debugging level, defaults to 256
+
+MAINTAINER Steve Kuznetsov <skuznets@redhat.com>
+
+ENV OPENLDAP_VERSION=2.4.41 \
+    HOME=/var/lib/openldap
+
+LABEL io.k8s.description="OpenLDAP is an open source implementation of the Lightweight Directory Access Protocol." \
+      io.k8s.display-name="OpenLDAP 2.4.41" \
+      io.openshift.expose-services="389:ldap,636:ldaps" \
+      io.openshift.tags="directory,ldap,openldap,openldap2441" \
+      io.openshift.non-scalable="true"
+
+# Install OpenLDAP Server
+RUN yum install -y openldap openldap-servers openldap-clients && \
+    yum clean all -y && \
+    mkdir -p /var/lib/ldap && \
+    mkdir -p /etc/openldap && \
+    chmod a+rw -R /var/lib/ldap && \
+    chmod a+rw -R /etc/openldap
+
+# Set OpenLDAP data and config directories in a data volume
+VOLUME ["/var/lib/ldap", "/etc/openldap"]
+
+# Expose default ports for ldap and ldaps
+EXPOSE 389 636
```

Comment:
@smarterclayton selinux is going to prevent this sort of thing at some point, right?